Source/Atomic/UI/UI.cpp

```diff
@@ -535,11 +539,52 @@ void UI::HandleUpdate(StringHash eventType, VariantMap& eventData)
         exitRequested_ = false;
         return;
     }
+    
+    mouseStayedTime += eventData[Update::P_TIMESTEP].GetFloat();
```

Comment:
I'm not sure that we need event for that, because we want have only one tooltip, so no need to  track hovered time in widget, we should have only one global hovered widget(multitouch?) and track time for it globally